### DISCUSSION TITLE
Zooplankton Bulk Metabarcoding

### DISCUSSION THREAD

**Discussion by marinednadude** [NOAA-Omics/NOAA-Omics#3] [2-Lab]
First discussion post!

Does anyone have recommendations for zooplankton bulk metabarcoding methods? Particularly interested in recommendations for primer sets. We are planning on running 5 marker sets on bulk homogenized zooplankton from 4 nets (200 µm vertical tow, 100 µm bongo, 200 µm bongo, and 335 µm bongo) taken from the West Coast Ocean Acidification Cruise. We are strongly leaning towards MiFish 12S, Leray CO1, and Kelly 16S for consistency with our previous work, but are open to suggestions for 2 additional marker sets.

Anyone have any suggestions/ideas?

MiDeca? A mollusk specific one for pteropods? Machida 18S? Some rad new primer set I've never come across?

**Comment by marinednadude** [comment]
@McAllister-NOAA Welcome to the discussion

**Comment by ksilnoaa** [comment]
I think we might try MiDeca near coral reefs soon, but with larger decapods in mind.

**Comment by marinednadude** [comment]
We  decided to focus on the ethanol preservative and test some bulk tissue sampling. We will largely follow Buchner et al. 2021 + following CalCOFI methods for getting zooplankton displacement.

We are going to use a hybrid approach from previously published protocols for extractions:
1. [Ethanol preservative](https://onlinelibrary.wiley.com/doi/pdf/10.1002/edn3.400)
2. [Buchner et al. 2021](https://www.google.com/url?q=https://www.researchgate.net/publication/353351387_Wet_grinding_of_invertebrate_bulk_samples_-_A_scalable_and_cost-efficient_protocol_for_metabarcoding_and_metagenomics&sa=D&source=docs&ust=1729789768805790&usg=AOvVaw0UtF0yQe_LrlxwzHc1fbb7)
3. [STREAM Benthic Invert. Metabarcoding](https://www.google.com/url?q=https://www.protocols.io/view/stream-benthic-metabarcoding-lab-protocol-draft-cyu4xwyw.html&sa=D&source=docs&ust=1729789768805824&usg=AOvVaw3ftNPgmwi7EAsBfZHKEIXc)
4. [Coral DNA Extraction](https://www.protocols.io/view/coral-dna-extraction-modified-dneasy-powersoil-pro-bww6pfhe.pdf)

As for markers we decided to go with
1. [Leray CO1 targeting Universal Metazoans](https://link.springer.com/content/pdf/10.1186/1742-9994-10-34.pdf)
2. [Machida 18S targeting Universal Metazoans](https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0046180&type=printable)
3. [Kelly 16S targeting Universal Metazoans](https://peerj.com/articles/2444/)
4. [MiDeca 16S targeting decapods](https://mbmg.pensoft.net/article/33835/download/pdf/) - (Thanks Katherine)
5. [MollCOI253 targeting mollusks](https://link.springer.com/article/10.1186/s12862-024-02265-8/tables/5)
